app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
```

Comment:
Fix code style. If you start arguments from a new line, then `) -> None` should be on a new line too

```
def __init__(
        self,
        comfort_class: int,
        clean_mark: int,
        brand: str
) -> None:
```